spec/app/domain/util/open_ssl/x509/request_spec.rb

```diff
@@ -0,0 +1,185 @@
+__END__
```

Comment:
I read about `__END__` but don't really understand why you're using it here. Mind explaining?

Reply:
Ha, just a sloppy way to comment it all out.  This file should be deleted now, which I will do.

Reply:
Aaah okay.